app-cdr/uif2iso/metadata.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <longdescription lang="en">
```

Comment:
Any specific reason you're removing this? Can't say it's very useful but I guess it doesn't seem to hurt either.